gradle.properties

```diff
@@ -0,0 +1,4 @@
+group = org.sonarsource.sonarqube
+version = 7.1-SNAPSHOT
+description = 'Open source platform for continuous inspection of code quality'
+sonarProjectName = SonarQube
```

Comment:
to remove ambiguity with native variable `name`, I suggest to rename `sonarProjectName` to `projectTitle`